src/Core/src/Handlers/Editor/EditorHandler.iOS.cs

```diff
@@ -11,7 +11,29 @@ public partial class EditorHandler : ViewHandler<IEditor, MauiTextView>
 	{
 		static readonly int BaseHeight = 30;
 
-		protected override MauiTextView CreatePlatformView() => new MauiTextView();
+		protected override MauiTextView CreatePlatformView()
+		{
+		var nativeEditor =	new MauiTextView();
+
+			if (DeviceInfo.Idiom == DeviceIdiom.Phone)
```

Comment:
Can this PR just be the code here? and then we can discuss the `AutoSize` changes here?

https://github.com/dotnet/maui/pull/4850

Reply:
Yes, updated the PR to only include the changes in the PlatformView. @PureWeen Could you review it again, please?